packages/tokamak/sdk/tasks/deposit-withdraw-ton.ts

```diff
@@ -0,0 +1,280 @@
+  process.env.L1_URL
+)
+const l2Provider = new ethers.providers.StaticJsonRpcProvider(
+  process.env.L2_URL
+)
+const l1Wallet = new ethers.Wallet(privateKey as BytesLike, l1Provider)
+const l2Wallet = new ethers.Wallet(privateKey as BytesLike, l2Provider)
+
+const erc20ABI = [
+  {
+    inputs: [
+      { internalType: 'address', name: '_spender', type: 'address' },
+      { internalType: 'uint256', name: '_value', type: 'uint256' },
+    ],
+    name: 'approve',
+    outputs: [{ internalType: 'bool', name: '', type: 'bool' }],
+    stateMutability: 'nonpayable',
+    type: 'function',
+  },
+  {
+    constant: true,
+    inputs: [{ name: '_owner', type: 'address' }],
+    name: 'balanceOf',
+    outputs: [{ name: 'balance', type: 'uint256' }],
+    type: 'function',
+  },
+  {
+    inputs: [{ internalType: 'uint256', name: 'amount', type: 'uint256' }],
+    name: 'faucet',
+    outputs: [],
+    stateMutability: 'nonpayable',
+    type: 'function',
+  },
+]
+
+const ETH = '0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000'
+
+const zeroAddr = '0x'.padEnd(42, '0')
+
+let TON = process.env.TON
+let addressManager = process.env.ADDRESS_MANAGER
+let l1CrossDomainMessenger = process.env.L1_CROSS_DOMAIN_MESSENGER
+let l1StandardBridge = process.env.L1_STANDARD_BRIDGE
+let optimismPortal = process.env.OPTIMISM_PORTAL
+let l2OutputOracle = process.env.L1_OUTPUT_ORACLE
+
+const updateAddress = async (hre) => {
+  if (TON === undefined || TON === '') {
+    const Deployment__TON = await hre.deployments.get('TON')
+    TON = Deployment__TON.address
+  }
+
+  if (addressManager === undefined || addressManager === '') {
+    const Deployment__AddressManager = await hre.deployments.get(
+      'AddressManager'
+    )
+    addressManager = Deployment__AddressManager.address
+  }
+
+  if (l1CrossDomainMessenger === undefined || l1CrossDomainMessenger === '') {
+    const Deployment__L1CrossDomainMessenger = await hre.deployments.get(
+      'L1CrossDomainMessengerProxy'
+    )
+    l1CrossDomainMessenger = Deployment__L1CrossDomainMessenger.address
+  }
+
+  if (l1StandardBridge === undefined || l1StandardBridge === '') {
+    const Deployment__L1StandardBridge = await hre.deployments.get(
+      'L1StandardBridgeProxy'
+    )
+    l1StandardBridge = Deployment__L1StandardBridge.address
+  }
+
+  if (optimismPortal === undefined || optimismPortal === '') {
+    const Deployment__OptimismPortal = await hre.deployments.get(
+      'OptimismPortalProxy'
+    )
+    optimismPortal = Deployment__OptimismPortal.address
+  }
+
+  if (l2OutputOracle === undefined || l2OutputOracle === '') {
+    const Deployment__L2OutputOracle = await hre.deployments.get(
+      'L2OutputOracleProxy'
+    )
+    l2OutputOracle = Deployment__L2OutputOracle.address
+  }
+}
+
+const depositTON = async (amount) => {
+  console.log('Deposit TON:', amount)
+  console.log('TON address:', TON)
+
+  const tonContract = new ethers.Contract(TON as string, erc20ABI, l1Wallet)
```

Comment:
We don't need to cast TON as a string because process.env.key always returns undefined or string, so we can use my suggestion to cast TON as a string

Same as other envs

Reply:
I updated!